### ISSUE TITLE
DAO experiment getters

### ISSUE THREAD

**Issue by petrmiko** [INCF/eeg-database#30]
There are multiple getters for experiments from person object and from experimentDao with on first sight unclear or even redundant purpose.

For example, in Person object, there is a method getExperiments which returns only some experiments - but not experiments which user owns, such experiments must be obtained by using method getExperimentsForOwnerId. Moreover, for experiments, where is user a subject there is method of getExperimentsForSubjectId. 

Bottom line - there is unclear when and where use which method in order to obtained desired data. This may not be a problem not only for experiments data type, but other types as well, this is only an example I've been facing recently.

**Comment by govind516** [comment]
@petrmiko, I understand the requirements and would like to work on them. I would require a guide to run the project with the necessary commands and installation setup.
Looking for your reply!

**Comment by petrmiko** [comment]
Hello @govind516, sorry but I cannot help here as I am no longer part of the project, really ran it last time in 2013 and not sure who maintains it now. The project seems not active anymore based on contributions. Cheers!
